common/common.go

```diff
@@ -425,8 +425,8 @@ var PermissionDeniedAPIError = status.Error(codes.PermissionDenied, "permission
 
 // Redis password consts
 const (
-	DefaultRedisInitialPasswordSecretName = "argocd-redis"
-	DefaultRedisInitialPasswordKey        = "auth"
+	DefaultRedisInitialCredentials    = "argocd-redis"
+	DefaultRedisInitialCredentialsKey = "auth"
```

Comment:
The old var names are so much more clear.... I'd rather try to find some way to get CodeQL to ignore the false positives

Reply:
They don't support annotations. It's just the words. I'm open to anything.

Reply:
You can mark an item as a false positive in repositories where you have certain permissions, but it has no impact on forks (connected or disconnected), and contributors won't be working from your repository's curated view, they'll work from the blank slate version. 

We can file [tickets against Code QL](https://github.com/github/codeql/issues/17574) – I've filed a bunch. But I don't see how to improve this one. Maybe we can ask them to improve the qhelp – I've done that a couple of times, but I'm not really sure what to say beyond a generic "please think before you file" which is too generic.

Reply:
Maybe best of both worlds: change the var names but add go doc strings to explicitly say that they're the secret name and the secret data field key?

Reply:
Sounds good to me. I'll try to do this today.